cactus/site.py

```diff
@@ -267,6 +267,10 @@ def build(self):
                 else:
                     os.remove(path)
 
+        # One test / use case depends on
+        # the pages being read from disk for every built
+        self._build_page_list()
```

Comment:
`tests.test_template_tags.TestMarkdown` changes files and builds an existing Site five times. So the page list has to be read for every built.